docs/mmctl_channel_move.rst

```diff
@@ -29,7 +29,8 @@ Options
 
 ::
 
-  -h, --help   help for move
+      --force   Remove members those are not member of target team before moving the channel.
```

Comment:
Looks like a typo.  Should be something like `Remove users that are not members of target team before moving the channel.`